<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>File System</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>701</width>
      <height>401</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="pathBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionCreateNew"/>
  </widget>
  <action name="actionBack">
   <property name="text">
    <string>后退</string>
   </property>
   <property name="toolTip">
    <string>后退</string>
   </property>
  </action>
  <action name="actionCreateNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
  </action>
  <action name="actionDeleteFile">
   <property name="text">
    <string>删除文件</string>
   </property>
  </action>
  <action name="actionDeleteFolder">
   <property name="text">
    <string>删除目录</string>
   </property>
  </action>
  <action name="readModelOpen">
   <property name="text">
    <string>以只读模式打开</string>
   </property>
  </action>
  <action name="writeModelOpen">
   <property name="text">
    <string>以写模式打开</string>
   </property>
  </action>
  <action name="actionChangeProperty">
   <property name="text">
    <string>修改属性</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
